modules/dxp/apps/search-experiences/search-experiences-blueprints-admin-web/src/main/resources/META-INF/resources/css/_clause_contributors_tab.scss

```diff
@@ -1,52 +1,50 @@
 .clause-contributors-tab {
-	.clause-contributor-table {
-		padding-top: calc(#{$pageToolbarHeight} + #{$managementBarHeight});
-
-		&.subnav-open {
-			padding-top: calc(
-				#{$pageToolbarHeight} + #{$managementBarHeight} + #{$subNavBarHeight}
-			);
-		}
-	}
-
-	.clause-content-shift {
-		margin-left: auto;
-		margin-right: calc(50% - #{$clauseMaxWidth} / 2);
-		max-width: $clauseMaxWidth;
-		transition: margin-right ease 0.2s;
-	}
-
-	.management-bar {
-		border-bottom: 1px solid $mainLighten74;
+	.clause-contributors-navbars {
```

Comment:
Is there a reason this is plural? Classnames/components are usually singular.

Reply:
Hm, interesting... I think it was because it included both the management bar and result bar. Should I just rename it as singular?

Reply:
I think singular makes sense since this "navbar" is a single bar which contains the management and result bar.